TheMET/TheMET/ViewControllers/CultureArtsViewController.swift

```diff
@@ -349,30 +203,18 @@ class CultureArtsViewController: UIViewController, UITableViewDelegate, UITableV
     }
     
     func tableView(_ tableView: UITableView, willDisplay: UITableViewCell, forRowAt: IndexPath) {
-        guard case .loaded(let arts) = self.contentStatus else {
+        guard case .loaded(let cultureArtStates) = self.contentStatus else {
             return
         }
-        let art = arts[forRowAt.row]
-        guard case .placeholder = art.artData,
-              !self.loadingArtIds.contains(art.artID) else {
-            return
-        }
-        self.loadingArtIds.append(art.artID)
-        self.loadArtCellData(artID: art.artID) { [weak self] artCellData in
-            if let artCellData = artCellData,
-               let contentStatus = self?.contentStatus,
-               case .loaded(var artCellDataList) = contentStatus,
-               let artCellDataIndex = artCellDataList.firstIndex(where: { $0.artID == art.artID}) {
-                artCellDataList[artCellDataIndex] = artCellData
-                self?.contentStatus = .loaded(artCellDataList)
-            }
-            self?.removeLoadingArtId(artId: art.artID)
-        }
+        let artState = cultureArtStates[forRowAt.row]
+        
+        self.viewModel?.tableViewWillDisplay(cultureArtState: artState)
     }
     
     //    UISearchBarDelegate
     func searchBar(_ searchBar: UISearchBar, textDidChange searchText: String) {
-        self.reloadCategory()
+        self.viewModel?.reloadCategory()
```

Comment:
- `reloadCategory` must be private and can't be executed from `ViewController` as this is not a UI-event of User-action
- How does `viewModel` know what Arts to search for if `searchText` is not used?